### PR TITLE
Jessie -> buster (which is the base image used in python 3.4)

### PR DESCRIPTION
**Pull request by pmauduit**
Reworked the docker setup, it is no longer possible to customize the targeted version though.

### CODE REVIEW

**Comment by pmauduit** [comment]
Tests are failing, hence the build result

**Comment by pvalsecc** [comment]
Yep: 
```
org.geoserver.wfs.WFSException: Unknown namespace [tiger]
```

File: acceptance_tests/Dockerfile [hunk 1]
```diff
@@ -2,13 +2,15 @@ FROM python:3.5
 
 RUN pip install pytest==3.0.2 requests==2.11.1 netifaces==0.10.5
 
+# DOCKER_VERSION is unused now
```
Comment (fvanderbiest) [inline acceptance_tests/Dockerfile]: It should probably be removed if unused

**Comment by yjacolin** [comment]
Data dir included in the image is empty: https://github.com/camptocamp/docker-geoserver/commit/0df844c44dfe08dbe0d33cafc206aad27a091912

I am going to add a data dir specific for testing.

**Pull request merged**